2022q4/pot.adoc

```diff
@@ -0,0 +1,23 @@
+=== Containers and FreeBSD: Pot, Potluck and Potman
+
+Links: +
+link:https://github.com/bsdpot[Pot organization on github] URL: link:https://github.com/bsdpot[https://github.com/bsdpot]
+
+Contact: Luca Pizzamiglio (Pot) <pizzamig@freebsd.org> +
+Contact: Stephan Lichtenauer (Potluck) <sl@honeyguide.eu> +
+Contact: Michael Gmelin (Potman) <grembo@freebsd.org>
+
+Pot is a jail management tool that link:https://www.freebsd.org/news/status/report-2020-01-2020-03/#pot-and-the-nomad-pot-driver[also supports orchestration through Nomad].
+
+During the last quarter, link:https://github.com/bsdpot/pot/releases/tag/0.15.4[pot 0.15.4] was released.
+It again contains a number of improvements like signing pot images as well as many bug fixes. Also, we welcome two new pot contributors link:https://github.com/zilti[@zilti] and link:https://github.com/reezer[@reezer].
+
+Additionally, there ist a new link:https://galaxy.ansible.com/zilti/pot[Ansible pot collection] available.
+
+Potluck aims to be to FreeBSD and pot what Dockerhub is to Linux and Docker: a repository of pot flavours and complete container images for usage with pot and in many cases Nomad.
+
+As you can see, we link:https://github.com/bsdpot/potluck/commits/master[had a busy quarter again], this time including improvements to the link:https://github.com/bsdpot/potluck/tree/master/nextcloud-nginx-nomad[Nextcloud] as well as link:https://github.com/bsdpot/potluck/tree/master/jitsi-meet[Jitsi] images.
+
+Also, Luca's EuroBSDCon 2022 talk link:https://www.youtube.com/watch?v=b0IB0mc2KTE[is now available on YouTube].
+
```

Comment:
Hi @hny-gd 

Thanks for preparing and sorry for the late reply.

There was one more story that is worth noting:

> We landed pot-based FreeBSD support for sccache-dist server (the server component for distributed compilation of rust and C++ using sccache) and it will be part of the upcoming sccache 0.4.0, see https://github.com/mozilla/sccache/pull/1184. Once released, this will become available through devel/sccache.
> 
> This means one can build rust projects on FreeBSD targeting a cluster of machines, something that could
> potentially integrated into poudriere as well.
>

Reply:
Thanks, added it.